src/XeroPHP/Remote/Query.php

```diff
@@ -105,16 +161,46 @@ public function offset($offset = 0) {
     }
 
     /**
+     * @param bool $includeArchived
+     * @return $this
+     */
+    public function includeArchived($includeArchived = true) {
```

Comment:
What's the point of this function accepting false as a parameter? Won't it just be left out when the params are built?  Another case for strict type checks for clarity on https://github.com/calcinai/xero-php/pull/52/files#diff-677c4a583ee02cfb477d433b7cbc2d55R224